app/src/main/java/org/fossasia/susi/ai/activities/ForgotPasswordActivity.java

```diff
@@ -95,7 +96,7 @@ public void signUp() {
         if(personalServer.isChecked()) {
             if(!CredentialHelper.checkIfEmpty(url,this) && CredentialHelper.isURLValid(url,this)) {
                 if (CredentialHelper.getValidURL(url,this) != null) {
-                    PrefManager.putBoolean("is_susi_server_selected", false);
+                    PrefManager.putBoolean(Constant.SUSI_SERVER, false);
                     PrefManager.putString("custom_server", CredentialHelper.getValidURL(url, this));
                 } else {
                     url.setError("Invalid URL");
```

Comment:
Hardcoded string here. Please don't leave these. It will be nice if all hardcoded strings are covered in one PR. If you need any help, let me know. I'll help you out. But let's cover all the hardcoded strings.